docs/concepts/clustering.rst

```diff
@@ -7,14 +7,14 @@ Data-Oriented Clustering
 Minimal Downtime Volume Migration
```

Comment:
thanks @sarum90 - yep, this PR just involves removing the Volumes doc. 

The changes to clustering.rst, (which was deemed relevant but out of date), will be covered in FLOC 953